javascript/javascript1/week2/task2.js

```diff
@@ -0,0 +1,32 @@
+
+function getEventWeekday(day){
+    let d = new Date();
```

Comment:
name the variables accordingly. don't use single letters it's should represent what you are assigning to it. it's very important in reading the code.